tests/api/test_clips.py

```diff
@@ -0,0 +1,54 @@
+import json
+
+import responses
+
+from twitch.client import TwitchClient
+from twitch.constants import BASE_URL
+from twitch.resources import Clip
+
+example_clip = {
+    "broadcast_id": "25782478272",
+    "title": "cold ace",
+    "tracking_id": "102382269",
+    "url": "https://clips.twitch.tv/OpenUglySnoodVoteNay?tt_medium=clips_api&tt_content=url"
+}
+
+example_clips = {'clips': [example_clip]}
+
+@responses.activate
+def test_get_slug():
```

Comment:
Can you change this function name to `test_get_by_slug`so it matches with the function name that it's testing please?